input/otlp/metrics_test.go

```diff
@@ -855,200 +855,222 @@ func TestConsumeMetricsDataStream(t *testing.T) {
 // code but does not test the correctness of the remapping libarary or if all
 // the metrics are remapped as it is within the scope of the remapping library.
 func TestConsumeMetricsWithOTelRemapper(t *testing.T) {
-	metrics := pmetric.NewMetrics()
-	rm := metrics.ResourceMetrics().AppendEmpty()
-	sm := rm.ScopeMetrics().AppendEmpty()
-
 	ts := time.Now().UTC()
 	startTs := ts.Add(-time.Hour)
 
-	// Configure scope for hostmetrics receiver
-	sm.Scope().SetName("otelcol/hostmetricsreceiver/load")
-
-	// Add a datapoint for a metric produced by hostmetrics receiver
-	metric := sm.Metrics().AppendEmpty()
-	metric.SetName("system.cpu.load_average.1m")
-	dp := metric.SetEmptyGauge().DataPoints().AppendEmpty()
-	dp.SetTimestamp(pcommon.NewTimestampFromTime(ts))
-	dp.SetDoubleValue(0.7)
-
-	// process metric
-	sm = rm.ScopeMetrics().AppendEmpty()
-	sm.Scope().SetName("otelcol/hostmetricsreceiver/process")
-	metric = sm.Metrics().AppendEmpty()
-	metric.SetName("process.memory.usage")
-	dp = metric.SetEmptySum().DataPoints().AppendEmpty()
-	dp.SetTimestamp(pcommon.NewTimestampFromTime(ts))
-	dp.SetStartTimestamp(pcommon.NewTimestampFromTime(startTs))
-	dp.SetIntValue(1024)
```

Comment:
[**For reviewers**] There should be no change in the tests. The tests just divides the load and the process scope into two separate tests.